pandas/core/frame.py

```diff
@@ -8201,7 +8201,7 @@ def stack(self, level: Level = -1, dropna: bool = True):
 
     def explode(
         self,
-        column: str | tuple | list[str | tuple],
+        column: Scalar | tuple | list[Scalar | tuple],
```

Comment:
we normally use Hashable for index labels. We also have the alias in pandas._typing `IndexLabel = Union[Hashable, Sequence[Hashable]]`

Reply:
Thx. #43834